velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -25,6 +25,30 @@ static int64_t toDate(std::string_view stringDate) {
   return date.days();
 }
 
+std::string TpchQueryBuilder::formatDateFilter(
+    std::string stringDate,
+    bool isDwrf,
+    std::string lowerBound,
+    std::string upperBound) {
+  std::string result;
+  if (lowerBound.size() && upperBound.size()) {
+    lowerBound = (isDwrf) ? (lowerBound) : (lowerBound + "::DATE");
+    upperBound = (isDwrf) ? (upperBound) : (upperBound + "::DATE");
+    result = stringDate + " BETWEEN " + lowerBound + " AND " + upperBound;
+  } else if (lowerBound.size()) {
+    lowerBound = (isDwrf) ? (lowerBound) : (lowerBound + "::DATE");
```

Comment:
drop parenthesis

```
auto suffix = isDwrf ? "" : "::DATE";

return fmt::format("{} > {}{}", stringDate, lowerBounds, suffix);
```

Reply:
Thanks for the feedback, this makes the code much more succinct and readable, I have modified it accordingly.